torch/_prims/nvfuser_executor.py

```diff
@@ -208,6 +213,9 @@ def nvfuser_execute(gm: GraphModule, *args, executor_parameters=None):
 
 class NvfuserPrimOperatorSupport(torch.fx.passes.operator_support.OperatorSupport):
     def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
+        # special case to stop lowering to nvprim when converting to an unsupported type
+        if node.op == "call_function" and node.name == "convert_element_type":
```

Comment:
This wouldn't work if we had multiple dtype conversions in the graph.
`node.name` refers to the variable name, we need to compare `node.target == torch.ops.prims.convert_element_type.default`:
```py
In [1]: import torch

In [2]: a = torch.randn(3, 3)

In [3]: def func(a):
   ...:     return (a.to(torch.float16), a.to(torch.float64))
   ...:

In [4]: from torch._prims.context import TorchRefsMode

In [5]: from torch.fx.experimental.proxy_tensor import make_fx

In [6]: with TorchRefsMode():
   ...:     make_fx(func)(a).graph.print_tabular()
   ...:
opcode         name                    target                              args                                               kwargs
-------------  ----------------------  ----------------------------------  -------------------------------------------------  --------
placeholder    a_1                     a_1                                 ()                                                 {}
call_function  convert_element_type    prims.convert_element_type.default  (a_1, torch.float16)                               {}
call_function  convert_element_type_1  prims.convert_element_type.default  (a_1, torch.float64)                               {}
output         output                  output                              ((convert_element_type, convert_element_type_1),)  {}
```

Reply:
Oops. good catch! it's concerning that this is not caught by our test. I'll update it along with python tests.

Reply:
BTW, we are inserting in the segmentation here, which I feel is a little bit odd and we are going to be skipping nvprim.convert_element_type instead.

I think you are suggesting moving this to `_is_node_supported_nvfuser` inside `context.py` instead.